src/helpers/utils.ts

```diff
@@ -117,3 +127,43 @@ export function filterProposals(space, proposal, tab) {
 
   return false;
 }
+
+export function scrollEndMonitor(fn) {
```

Comment:
Can you remove this? I think that come from older code, we use "useScrollMonitor" composable now.